src/systemverilog/systemverilog.contribution.ts

```diff
@@ -8,7 +8,16 @@ import { registerLanguage } from '../_.contribution';
 
 registerLanguage({
 	id: 'systemverilog',
-	extensions: ['.sv', '.svh', '.v'],
+	extensions: ['.sv', '.svh'],
 	aliases: ['SV', 'sv'],
```

Comment:
in the aliases can we also add SystemVerilog and systemverilog

same for Verilog/verilog